pdcupdater/utils.py

```diff
@@ -475,6 +481,18 @@ def interesting_tags():
     return stable_tags + [rawhide_tag()]
 
 
+def interesting_container_tags():
+    """ Returns a list of "interesting tags" relevant to containers.
+
+    Eventually, we should query PDC itself to figure out what tags we should be
+    concerned with.
+    """
+    return ['f24-docker']  # This is all we're producing for now...
+    #tags = interesting_tags()
```

Comment:
Yeah, I'll keep this one as I know I need to double back on it once we have a good place to look for the tags.